pybandits/base.py

```diff
@@ -45,3 +51,42 @@ class PyBanditsBaseModel(BaseModel, extra="forbid"):
     """
     BaseModel of the PyBandits library.
     """
+
+    def _apply_version_adjusted_method(self, v2_method_name: str, v1_method_name: str, **kwargs) -> Any:
+        """
+        Apply the method with the given name, adjusting for the pydantic version.
+
+        Parameters
+        ----------
+        v2_method_name : str
+            The method name for pydantic v2.
+        v1_method_name : str
+            The method name for pydantic v1.
+        """
+        if pydantic_version == "1":
```

Comment:
As in pydantic_version_compact.py. You can use a literal do define version, instead of hardcoded string.